### PR TITLE
Helm artifact setup docs

### PR DESCRIPTION
**Pull request by shubhamc183**
Solves: https://github.com/spinnaker/spinnaker.github.io/issues/1994

### CODE REVIEW

File: setup/artifacts/helm.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: single
+title:  "Configure Helm Artifact account"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker stages that read data from artifacts can read Helm provider directly. The provider can be Artifactory like [Nexus](https://help.sonatype.com/repomanager3/formats/helm-repositories), [JFrog](https://jfrog.com/integration/helm-repository/), or [Chartmuseum](https://chartmuseum.com/).
```
Comment (dorbin) [inline setup/artifacts/helm.md]:
```suggestion
Spinnaker stages that consume artifacts can read from a Helm provider directly. The provider can be an Artifactory like [Nexus](https://help.sonatype.com/repomanager3/formats/helm-repositories), [JFrog](https://jfrog.com/integration/helm-repository/), or [Chartmuseum](https://chartmuseum.com/).
```

**Review by dorbin — COMMENTED** [review]
@shubhamc183 , you say "If the files are hidden by basic auth..." Do we support setting this up for anything other than basic auth?

**Comment by dorbin** [comment]
Oh, also, @shubhamc183 can you add an entry for this doc in `/_data/navigation.yml`? Thanks!

**Comment by shubhamc183** [comment]
Added entry for this doc in ` /_data/navigation.yml`.

Also, for the `helm` artifact only basic auth is present.

https://spinnaker.io/reference/halyard/commands/#hal-config-artifact-helm-account-add

File: setup/artifacts/helm.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+title:  "Configure Helm Artifact account"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker stages that consume artifacts can read from a Helm provider directly. The provider can be an Artifactory like [Nexus](https://help.sonatype.com/repomanager3/formats/helm-repositories), [JFrog](https://jfrog.com/integration/helm-repository/), or [Chartmuseum](https://chartmuseum.com/).
+
+If the files are hidden behind basic auth, you can configure an artifact account with the needed credentials to read your artifact.
```
Comment (dorbin) [inline setup/artifacts/helm.md]:
```suggestion
If the files are hidden behind basic auth, you can configure an artifact account with the needed credentials to read your artifacts. Basic auth is the only authentication mechanism supported for accessing a Helm artifact account.
```

**Review by dorbin — COMMENTED** [review]
Thanks, @shubhamc183 . I've added a wording suggestion based on the fact that we only support basic auth.

**Review by dorbin — APPROVED** [review]

**Review by ezimanyi — APPROVED** [review]

**Pull request merged**